### PR TITLE
Coverage fix

### PR DESCRIPTION
**Pull request by ghost**
- fixes a frontend issues by removing wrong values pushed to frontend
- is stiched together on the frontend

### CODE REVIEW

**Comment by ghost** [comment]
WARNING: Discarding https://files.pythonhosted.org/packages/99/5a/fc440eb5fffb5346e61a38b49991aa552e4b8b31e8493a101d2833ed1e19/pysam-0.16.0.1.tar.gz#sha256=d428a9768691d5ea3c28cc52a949c920ae691aa4c110a8b7328dc4d165ef1ad6 (from https://pypi.org/simple/pysam/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement pysam~=0.16.0.1 (from versions: 0.4, 0.4.1, 0.7.6, 0.7.7, 0.7.8, 0.8.0, 0.8.1, 0.8.2, 0.8.2.1, 0.8.3, 0.8.4, 0.9.0, 0.9.1, 0.9.1.1, 0.9.1.2, 0.9.1.3, 0.9.1.4, 0.10.0, 0.11, 0.11.1, 0.11.2, 0.11.2.1, 0.11.2.2, 0.12, 0.12.0.1, 0.13, 0.14, 0.14.1, 0.15.0, 0.15.1, 0.15.2, 0.15.3, 0.15.4, 0.16.0, 0.16.0.1, 0.17.0, 0.18.0)
ERROR: No matching distribution found for pysam~=0.16.0.1
Error: Process completed with exit code 1.

File: src/epivizFileParser/BamFile.py [hunk 1]
```diff
@@ -80,6 +80,13 @@ def getRange(self, chr, start, end, bins=2000, zoomlvl=-1, metric="AVG", respTyp
 
             chrTemp = startTemp = endTemp = valueTemp = None
             for x in iter:
+
+                if not x.reference_pos >= start or not x.reference_pos <= end:
+                        continue
+                
+                if x.reference_pos == end and valueTemp is not None:
+                    result.append((chrTemp, startTemp, end, valueTemp))
```
Comment (jkanche) [inline src/epivizFileParser/BamFile.py:88]: This would potentially add the last interval twice... needs a `continue` here

Comment (ghost) [reply src/epivizFileParser/BamFile.py:88]: it's weird how this never caused any issue in testing

but I added it since it makes sense to

**Comment by ghost** [comment]
tox passed locally

**Pull request merged**